include/ginkgo/core/solver/lower_trs.hpp

```diff
@@ -121,8 +111,11 @@ class LowerTrs : public EnableLinOp<LowerTrs<ValueType, IndexType>>,
         /**
          * Number of right hand sides.
          *
-         * @note: This value must be same as to be passed to the b vector of
-         * apply.
+         * @note: This value is currently a dummy value which is not used by the
+         * analysis step. It is possible that future algorithms (cusparse
+         * csrsm2) make use of the number of right hand sides for a more
+         * sophisticated implementation. Hence this parameter is left here. But
+         * currently, there is no need to use it.
```

Comment:
Could you make this align with the `@note`? Also, I don't think the colon is necessary (I also don't know if it causes problems either).